man/epi_slide_opt.Rd

```diff
@@ -59,44 +59,44 @@ these args via \code{...} will cause an error. If \code{f} is a \code{slider} fu
 it is automatically passed the data \code{x} to operate on, and number of
 points \code{before} and \code{after} to use in the computation.}
 
-\item{before, after}{How far \code{before} and \code{after} each \code{ref_time_value} should
-the sliding window extend? At least one of these two arguments must be
-provided; the other's default will be 0. The accepted values for these
-depend on the type of the \code{time_value} column:
-\itemize{
-\item if it is a Date and the cadence is daily, then they can be integers
-(which will be interpreted in units of days) or difftimes with units
-"days"
-\item if it is a Date and the cadence is weekly, then they must be difftimes
-with units "weeks"
-\item if it is an integer, then they must be integers
-}
+\item{before, after}{Deprecated. Use \code{.n} instead.}
```

Comment:
issue: looks like `@template` usage is making this documentation inaccurate.  Ideas: either
- ensure both `epi_slide` and `epi_slide_{opt,mean,sum}` have appropriate parameter documentation --- this means they shouldn't use the same `@template` to document `before`, `after`, `.n`, `.align`, maybe some others
- change `epi_slide_opt` to use `.n` and `.align`, or
- change `epi_slide_opt` to use `n` and `align` (like e.g., `slice_head`) PLUS fix the `@template` issue

Reply:
Updated all the documentation that I could find. I'm sure there will be things I missed. Will fix issues as I spot them on the site in the documentation pass coming up.